.gitignore

```diff
@@ -17,9 +17,9 @@
 .env.test.local
 .env.production.local
 package-lock.json*
-yarn-lock.json*
+package.json
```

Comment:
we have to check in the package.json file because this is required by npm when someone else clones down the repo. you will have to remove this line